src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -830,6 +875,15 @@ internal void ChangeMutationPoints(int change)
         MutationPoints = (MutationPoints + change).Clamp(0, Constants.BASE_MUTATION_POINTS);
     }
 
+    /// <summary>
+    ///   Called by PlayerMicrobeEditorInput
```

Comment:
Is this comment accurate? This looks to be called from within this class. And summary should be what this does, not who calls this.